packages/dx-react-chart/src/plugins/basic-data.tsx

```diff
@@ -1,9 +1,8 @@
 import * as React from 'react';
-import * as PropTypes from 'prop-types';
 import { Plugin, Getter } from '@devexpress/dx-react-core';
-import { defaultDomains } from '@devexpress/dx-chart-core';
+import { defaultDomains, DataItems } from '@devexpress/dx-chart-core';
 
-export const BasicData = ({ data }) => (
+export const BasicData = ({ data }: { data: DataItems }) => (
```

Comment:
It should something like `BasicData: React.SFC<BasicDataProps> = ({ data }) => { ... }`.